traversal/procedure/GraphProcedure.java

```diff
@@ -60,14 +64,6 @@ private GraphProcedure(ProcedureVertex<?, ?>[] vertices) {
         this.vertices = vertices;
     }
 
-    public static GraphProcedure create(GraphPlanner planner) {
```

Comment:
move into `Builder`

Reply:
Following a discussion we decided to move the new `create` methods back here.